extension/modules/personas.js

```diff
@@ -1178,7 +1178,7 @@ var PersonaController = {
 
         item.setAttribute("class", "menuitem-iconic");
         item.setAttribute("label", this._strings.get("viewMoreFrom", [category]));
-        item.addEventListener("command",function(event){PersonaController.openURLInTab(this.getAttribute('href'),event)},true);
+        item.addEventListener("command", function(event) { PersonaController.openURLInTab(this.getAttribute('href'), event) }, true);
```

Comment:
Code formatting should be consistent. Anonymous functions should have the same style everywhere.